spec/views/early_allocations/show.html.erb_spec.rb

```diff
@@ -0,0 +1,116 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe "early_allocations/show", :early_allocation, type: :view do
+  context 'when reviewing an early assessment' do
+    before do
+      assign(:case_information, build(:case_information))
+      assign(:offender, build(:offender, sentence: attributes_for(:sentence_detail)))
+      assign(:early_assignment, create(:early_allocation, :eligible, created_at: '05/11/2021', created_by_firstname: 'Olivia', created_by_lastname: 'Mann'))
+
+      render
+    end
+
+    let(:page) { Nokogiri::HTML(rendered) }
```

Comment:
It looks like `let(:page)` this line could be DRYed out by pulling it to the top of this spec.

I think it's used by most of the `context` blocks – so it should only need to be declared once in the top-level `RSpec.describe` block.